src/components/manage/Blocks/Block/Edit.jsx

```diff
@@ -137,88 +122,60 @@ class Edit extends Component {
     const blockHasOwnFocusManagement =
       blocks.blocksConfig?.[type]?.['blockHasOwnFocusManagement'] || null;
 
-    return (
-      <div className={`ui drag block inner ${type}`}>
-        {Block !== null ? (
-          <div
-            role="presentation"
-            onClick={(e) => {
-              const isMultipleSelection = e.shiftKey || e.ctrlKey || e.metaKey;
```

Comment:
There is code that was added to Volto after I've forked the blocks engine. This needs to be updated according to latest Volto master functionality. In particular here it's missing the multi-block copy/paste.

Reply:
Aha! Sorry I missed the changes from master Edit block. The test should pass now. Thanks

Reply:
and the diff is much smaller :)

Reply:
Sorry to do this to you, but there's a few other places where you have to sync API changes. Basically everything in the blocks helpers needs to be compared with the code that you've ripped out in Form.jsx, as there has been updates in Form.jsx. 

https://github.com/plone/volto/blob/master/src/helpers/Blocks/Blocks.js

I think all changes in Form.jsx after September 2020 needs to be synchronized to the Blocks.js helper and BlockForm.